### PR TITLE
fix golint warning

### PR DESCRIPTION
**Pull request by Icedroid**
- fix code by golint warning

### CODE REVIEW

File: pkg/builder/volbuilder/build.go [hunk 1]
```diff
@@ -178,6 +178,7 @@ func (b *Builder) WithSnapshot(snap string) *Builder {
 	return b
 }
 
+// WithPoolName sets Pool name for creating clone volume
```
Comment (pawanpraka1) [inline pkg/builder/volbuilder/build.go:181]: n_: for creating volume.

File: pkg/builder/volbuilder/build.go [hunk 2]
```diff
@@ -192,7 +193,8 @@ func (b *Builder) WithPoolName(pool string) *Builder {
 	return b
 }
 
-func (b *Builder) WithNodename(name string) *Builder {
+// WithNodeName sets NodeID for creating clone volume
```
Comment (pawanpraka1) [inline pkg/builder/volbuilder/build.go:196]: n_: creating the volume

File: pkg/builder/volbuilder/build.go [hunk 3]
```diff
@@ -223,6 +225,7 @@ func (b *Builder) WithLabels(labels map[string]string) *Builder {
 	return b
 }
 
+// WithFinalizer sets Finalizer name creating clone volume
```
Comment (pawanpraka1) [inline pkg/builder/volbuilder/build.go:228]: n_: creating the volume.

File: pkg/driver/controller.go [hunk 4]
```diff
@@ -127,6 +129,7 @@ func CreateZFSVolume(req *csi.CreateVolumeRequest) (string, error) {
 	return selected, nil
 }
 
+// CreateZFSClone create an clone of zfs volume
```
Comment (pawanpraka1) [inline pkg/driver/controller.go:132]: n_: create a clone

File: pkg/zfs/zfs_util.go [hunk 5]
```diff
@@ -48,6 +48,7 @@ const (
 	VOLTYPE_ZVOL    = "ZVOL"
 )
 
+// PropertyChanged return whether volume is changed
```
Comment (pawanpraka1) [inline pkg/zfs/zfs_util.go:51]: n_: volume property.

**Comment by pawanpraka1** [comment]
@Icedroid can you run `make format` in repo directory. it will format/align the code. After this you can check the `git status` to get the list of modified files. Please push those files in this PR.

**Comment by Icedroid** [comment]
I run make in my mac is failed.
❯ make
--> Running go fmt
--> Running go test
# github.com/openebs/zfs-localpv/pkg/driver
pkg/driver/agent.go:342:32: invalid operation: int64(sfs.Blocks) * sfs.Bsize (mismatched types int64 and uint32)
pkg/driver/agent.go:343:42: invalid operation: int64(sfs.Blocks - sfs.Bfree) * sfs.Bsize (mismatched types int64 and uint32)
pkg/driver/agent.go:344:32: invalid operation: int64(sfs.Bavail) * sfs.Bsize (mismatched types int64 and uint32)
ok      github.com/openebs/zfs-localpv/pkg/client/k8s/v1alpha1  1.512s
ok      github.com/openebs/zfs-localpv/pkg/common/kubernetes/client     2.571s
ok      github.com/openebs/zfs-localpv/pkg/usage        1.718s
make: *** [test] Error 1

**Comment by pawanpraka1** [comment]
@Icedroid if you have changed crd(comment or anything), then run `make manifests` to generate the crds as they are auto generated. So you need to do two things

1. make format <==== for formatting 
2. make manifests  <==== for crd changes

And add the unstaged files and push. 

Also need a favor, can you raise a separate PR to help improve the contrubuting document. Please add the missing details so that no one else faces the same issue. 

https://github.com/openebs/zfs-localpv/blob/master/CONTRIBUTING.md#pull-request-checklist

**Comment by pawanpraka1** [comment]
@Icedroid There are still few linter errors, you want to take a look?
```
$ golint pkg/... tests/...
pkg/apis/openebs.io/zfs/v1alpha1/zfssnapshot.go:49:6: exported type SnapStatus should have comment or be unexported
pkg/apis/openebs.io/zfs/v1alpha1/zfsvolume.go:201:6: exported type VolStatus should have comment or be unexported
pkg/client/k8s/v1alpha1/clientset.go:32:18: exported func Clientset returns unexported type *v1alpha1.clientset, which can be annoying to use
pkg/client/k8s/v1alpha1/configmap.go:38:40: exported func ConfigMap returns unexported type *v1alpha1.configmap, which can be annoying to use
pkg/client/k8s/v1alpha1/dynamic.go:32:16: exported func Dynamic returns unexported type *v1alpha1.dynamic, which can be annoying to use
pkg/client/k8s/v1alpha1/namespace.go:37:18: exported func Namespace returns unexported type *v1alpha1.namespace, which can be annoying to use
pkg/client/k8s/v1alpha1/node.go:36:1: exported function Node should have comment or be unexported
pkg/client/k8s/v1alpha1/node.go:36:13: exported func Node returns unexported type *v1alpha1.node, which can be annoying to use
pkg/client/k8s/v1alpha1/node.go:45:9: if block ends with a return statement, so drop this else and outdent its block
pkg/client/k8s/v1alpha1/node.go:55:9: if block ends with a return statement, so drop this else and outdent its block
pkg/client/k8s/v1alpha1/node.go:66:9: if block ends with a return statement, so drop this else and outdent its block
pkg/client/k8s/v1alpha1/node.go:77:9: if block ends with a return statement, so drop this else and outdent its block
pkg/client/k8s/v1alpha1/node.go:88:9: if block ends with a return statement, so drop this else and outdent its block
pkg/client/k8s/v1alpha1/resource.go:80:66: exported func Resource returns unexported type *v1alpha1.resource, which can be annoying to use
pkg/usage/const.go:1:1: package comment should be of the form "Package usage ..."
pkg/usage/const.go:37:2: comment on exported const EventLabelNode should be of the form "EventLabelNode ..."
pkg/usage/const.go:39:2: exported const EventLabelCapacity should have comment (or a comment on this block) or be unexported
pkg/usage/versionset.go:48:19: exported func NewVersion returns unexported type *usage.versionSet, which can be annoying to use
pkg/zfs/xfs_util.go:72:6: func xfsGenerateUuid should be xfsGenerateUUID
pkg/zfs/zfs_util.go:29:2: don't use ALL_CAPS in Go names; use CamelCase
pkg/zfs/zfs_util.go:30:2: don't use ALL_CAPS in Go names; use CamelCase
pkg/zfs/zfs_util.go:47:2: don't use ALL_CAPS in Go names; use CamelCase
pkg/zfs/zfs_util.go:48:2: don't use ALL_CAPS in Go names; use CamelCase
tests/utils.go:22:2: should not use dot imports
tests/utils.go:23:2: should not use dot imports
tests/utils.go:175:1: exported function VerifyZFSVolume should have comment or be unexported
tests/utils.go:200:1: exported function VerifyZFSVolumePropEdit should have comment or be unexported
tests/pod/pod.go:27:6: type name will be used as pod.PodList by other packages, and that stutters; consider calling this List
tests/pvc/persistentvolumeclaim.go:33:6: type name will be used as pvc.PVCList by other packages, and that stutters; consider calling this List
```

**Comment by pawanpraka1** [comment]
@Icedroid  can you add a changelog also https://github.com/openebs/zfs-localpv/blob/master/CONTRIBUTING.md#adding-a-changelog.  See the example here :- https://github.com/openebs/zfs-localpv/blob/master/changelogs/unreleased/121-pawanpraka1

**Pull request merged**